python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1967,12 +1914,20 @@ def test_reset_index_inplace(pdf, gdf, drop):
     assert_eq(pdf, gdf)
 
 
+@pytest.mark.xfail(
+    reason="Individual Columns in a DataFrame are "
+    "stored as Columns, not Series"
+)
```

Comment:
I don't understand why we need to xfail this now.

Reply:
We discussed offline and some of these tests we just needed to get rid of, as we don't represent a DF as a dict of Series anymore.

There's no way for downstream changes to the Series metadata of `df['a']` to affect the dataframe `df` itself.